SCF/SCF-3/HexCalc.java

```diff
@@ -0,0 +1,14 @@
+
+public class HexCalc {

```

Comment:
Class HexCalc is not created according to the assignment requirement. According to the assignment, HexCalc is the class which contains all the arithmetic and conversion method. Please read the assignment carefully before start coding.